hazelcast-client/src/main/java/com/hazelcast/client/config/ClientConfig.java

```diff
@@ -764,10 +764,10 @@ public ClientConfig setUserCodeDeploymentConfig(ClientUserCodeDeploymentConfig u
     /**
      *
      * @param mapName The name of the map for which the query cache config is to be returned.
-     * @param queryCacheName The name of the query cache config.
+     * @param userGivenCacheName The name of the query cache config.
      * @return The query cache config. If the config does not exist, it is created.
      */
-    public QueryCacheConfig getOrCreateQueryCacheConfig(String mapName, String queryCacheName) {
+    public QueryCacheConfig getOrCreateQueryCacheConfig(String mapName, String userGivenCacheName) {
```

Comment:
why not call this 'cacheName' since it is the name of the cache. What is the added value of calling it the 'userGivenCacheName'?

If you need to use a more technical identifier internally to identify a map/cache.. I would call that 'mapId' or something

Reply:
fixed